dist/data.js

```diff
@@ -16594,7 +16731,9 @@ const THE_OBJ = (function () {
             "of": null,
             "of_ptr": false,
             "type": "method",
-            "params": [{"name": "number", "type": 1, "is_ptr": false}, {"name": "number", "type": 1, "is_ptr": false}]
+            "params": [{"name": "player1", "type": 1, "is_ptr": false}, {"name": "player2", "type": 1, "is_ptr": false}],
+            "description_en": "Returns true if <tt>player1</tt> and <tt>player2</tt> cannot attack each other, or false otherwise.",
```

Comment:
Have you checked this? I run with wine so there might be something wrong but I get false even when ceasing fire between the players. In my case, I need to have all shared: view, control, supply, cease fire and allied victory. Please revise this. Also detail how _not-symmetrical_ relations affect it (some methods behave weird with these).

Reply:
I only tested this with allied victory enabled. After doing more testing, it seems like both players must have Allied victory set and symmetrical cease fire for it to return true. Will update to account for this.

For future reference: the functions `_PlayersAlly` sets the allied victory flag between two players and `_PlayersMakeEnemies` clears the flag. Will eventually document these once I finish fixing all the current issues.

Reply:
Also the Allied victory flag is different from the other flags in that it seems to be symmetric (`_PlayersAlly(1, 2) <=> _PlayersAlly(2, 1)`)